x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/single_page_layout/index.tsx

```diff
@@ -27,7 +27,11 @@ import { useCancelAddPackagePolicy, useOnSaveNavigate } from '../hooks';
 import type { CreatePackagePolicyRequest } from '../../../../../../../common/types';
 
 import { splitPkgKey } from '../../../../../../../common/services';
-import { dataTypes, FLEET_SYSTEM_PACKAGE } from '../../../../../../../common/constants';
+import {
+  dataTypes,
+  FLEET_SYSTEM_PACKAGE,
+  HIDDEN_API_REFERENCE_PACKAGES,
```

Comment:
I cannot find this constant in the code. Is it a static list?

Reply:
oops I forgot to commit a file, should be fixed :)